QUEST_LV_0100.tsv

```diff
@@ -3377,7 +3377,7 @@ QUEST_LV_0100_20150317_003376	About the possessed soul
 QUEST_LV_0100_20150317_003377	$Truth of the Suspicious Seal Stone (2)
 QUEST_LV_0100_20150317_003378	I will release the soul by defeating Stone Whale
 QUEST_LV_0100_20150317_003379	{memo X}NPCChat/FTOWER45_SQ_04 /Now I am really free
-QUEST_LV_0100_20150317_003380	Goddess Tower
+QUEST_LV_0100_20150317_003380	Goddess' Tower
```

Comment:
This is a quest title; it's basically the quest for saving Goddess Gabija in Mage Tower.
I believe b/c Flurry died, Gabija owns the Mage Tower. Thus, I'm trying to directly convey that the Tower is in possession by Gabija. Tower of Goddess and Tower of the Goddess would work too, I guess. But people can also infer from those title names that the Tower is simply devoted to the Goddess and might be generic sounding?